docroot/themes/humsci/su_humsci_theme/patterns/molecules/featured-card/featured-card.html.twig

```diff
@@ -0,0 +1,40 @@
+{% set attributes = attributes.addClass('vertical-card') %}
```

Comment:
This class should be the same as the pattern id

Reply:
Word. Good catch.